python/triton_kernels/triton_kernels/matmul_details/opt_flags_details/opt_flags_amd.py

```diff
@@ -23,11 +23,17 @@ def compute_block_nk(n, block_m, grid_m, num_xcds, lhs_dtype, rhs_dtype, precisi
     if get_cdna_version() == 4 and block_m == 128:
         block_n = 512
 
+    if get_rdna_version() != -1 and block_m == 64:
```

Comment:
Let's make it explicit by checking `get_rdna_version() in (3, 4)` to avoid suddenly this kicks in if somebody adds other version support for example. Similarly for other places.